### PR TITLE
Popup Feature

### PR DESCRIPTION
**Pull request by kaizipaul**
In this PR:
- Added markup for the popup window with HTML file
- Added styling to the popup window using CSS file
- Added functionality to add the work grid using Javascript
- Used javascript objects to store data for the work grid cards
- Added event listeners using javascript to enable the popup windows

**Thank you for reviewing my PR**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,7 +1,57 @@
 /* eslint-disable linebreak-style */
+const cardsData = [
+  {
+    id: 1,
+    title: 'Card 1',
+    image: './assets/images/card-image.png',
+    description:
+        "1 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
```
Comment (CollinsTatang) [inline script.js]: - [x] Please your project object misses the following information: technologies, link to live version, and link to the project source code. 
### **The required object structure**
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @kaizipaul, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -1,7 +1,68 @@
 /* eslint-disable linebreak-style */
+const cardsData = [
+  {
+    id: 1,
+    title: 'Card 1',
+    image: './assets/images/card-image.png',
+    description:
+        "1 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['CSS', 'Javascript', 'Ruby'],
+    link: 'link_to_live_version.com',
```
Comment (BertrandConxy) [inline script.js:10]: - [x] [OPTIONAL] It would be good if you add the link to the source code too 😉

**Review by BertrandConxy — APPROVED** [review]
Hi @kaizipaul , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 

- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**